runtime/src/lib.rs

```diff
@@ -13,6 +13,7 @@ pub use frame_support::traits::EqualPrivilegeOnly;
 use frame_support::{
 	traits::{ConstU32, ConstU8, Currency, U128CurrencyToVote},
 	weights::{WeightToFeeCoefficient, WeightToFeeCoefficients, WeightToFeePolynomial},
+	PalletId,
```

Comment:
You will have to bump the version. See the failed CI job.

In general any changes to `runtime/` will require a version bump.